include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -10,20 +10,43 @@
 #include <limits>
 #include <stdexcept>
 
-
+/**
```

Comment:
I think this comment would bind to `namespace SuperiorMySqlpp`.

Reply:
OK, changed to ordinary comment. In order to work properly with doxygen, `namespace Converters` should be on separate line and everything below would have to be indented